sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -11,9 +11,18 @@
 Note: Below breaking changes only affect the version `1.5.0-beta.1`.
 
 - Changed `listConfigurationSettingsForSnapshot` method's `SettingFields[] fields` parameter type to `List<SettingFields> fields`.
-- Renamed `ConfigurationSettingSnapshot` to `ConfigurationSettingsSnapshot`.
-- Renamed `ConfigurationSettingSnapshot`'s property name, `size` to `sizeInBytes`.
-
+- Renamed `ConfigurationSettingSnapshot` to `ConfigurationSnapshot`.
+- Renamed `ConfigurationSettingSnapshot`'s properties name,
+  - `size` to `sizeInBytes`,
+  - `compositionType` to `snapshotComposition`.
+- Added `azure-core-experimental` as a dependency and replaced usage of `CreateSnapshotOperationDetail` by `PollResult` 
```

Comment:
Is `PollResult` something we're planning to move to `azure-core` before this needs to be released GA?

@srnagar

Reply:
Yep. @weidongxu-microsoft will have more context on it. Current plan is release next beta of AppConfig with `azure-core-experiemental` beta dependency. But it has to have it in Core before/on Nov Azure Core version. AppConfig will GA in November.

Reply:
@alzimmermsft 

Context in PR https://github.com/Azure/azure-sdk-for-java/pull/34525

However, the origin purpose of it (aka `PollResult` and `OperationLocationPollingStrategy`) is for the new LRO design (by Jeffrey around 2022).
Major concern of it being in experiential is that TypeSpec Azure.Core is still preview. https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-core/lib/foundations.tsp#L37-L51

Here it seems the `PollResult` is just used as a model in polling.

Reply:
PR https://github.com/Azure/azure-sdk-for-java/pull/36993

Reply:
@alzimmermsft since we are targeting to include `PollResult` type in Azure Core November release. We can include this type (even if it could be renamed or minor changes) and test it out before AppConfig GA in Nov).
What do you think?

Reply:
Yeah, if this is planned to be moved to azure-core before the GA of AppConfiguration this is fine to do.